cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/VolumeParameterConfig.java

```diff
@@ -60,4 +62,12 @@ public String toString() {
                 + ", maximumNumber=" + maximumNumber
                 + '}';
     }
+
+    private Integer calculateSupportedMaximumNumber(Integer maximumNumber) {
+        Integer result = maximumNumber;
+        if (maximumNumber > VolumeParameterConstants.MAXIMUM_NUMBER_OF_VOLUMES) {
+            result = VolumeParameterConstants.MAXIMUM_NUMBER_OF_VOLUMES;
+        }
+        return result;
```

Comment:
Perhaps simplify to `return Math.min(VolumeParameterConstants.MAXIMUM_NUMBER_OF_VOLUMES, maximumNumber)`?

Reply:
I can use this instead of the if condition, but I think that's a style related decision and for me the condition emphasise more that the number is overridden and easier to read then a Math expression 😄